Remotion/Validation/UnitTests/Validators/NotEmptyCollectionValidatorTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using Remotion.Validation.Implementation;
+using Remotion.Validation.Validators;
+
+namespace Remotion.Validation.UnitTests.Validators
+{
+  [TestFixture]
+  public class NotEmptyCollectionValidatorTest : ValidatorTestBase
```

Comment:
pelase add tests for all collection property types. Right now, we don't have full coverage. We should also add an explicit test for empty IEnumerable<object> to show that that's ignored.

Reply:
I added the tests, but I needed some test classes in order to separate the three interfaces; I doubt that the benefits of those tests are worth the effort, because the code we test here is trivial.